charts/osm/crds/meshconfig.yaml

```diff
@@ -117,6 +117,34 @@ spec:
                       description: True for allowing traffic to flow between client and service pods within the mesh without SMI traffic policies, i.e. no traffic policy enforcement in the mesh. If set to false, enables deny-all traffic policy in mesh i.e. an SMI Traffic Target is necessary for services to communicate.
                       type: boolean
                       default: false
+                    inboundExternalAuthorization:
+                      description: Configures additional external authorization for inbound and ingress connections.
```

Comment:
What does additional imply here?

Reply:
i think I had in mind RBAC or other things we might do, if a user wants to. I'll remove the `additional`.